deploy/operator.yaml

```diff
@@ -161,6 +166,7 @@ metadata:
 data:
   node_network_state_refresh_interval: "5"
   interfaces_filter: "veth*"
+  webhook_port: "10443"
```

Comment:
This is something coupled with deployment, not with actual configuration of the application. The port should be setup by the person who sets up the cluster and knows what ports can collide. nmstate configuration contains stuff that adjusts behavior of the app and is important to the user.

I think we should keep it solely in the DaemonSet definition.